pages/api-reference/index.mdx

```diff
@@ -55,6 +55,29 @@ Dwolla recommends building a mechanism into your application for `429 Too Many R
 
 If you believe that your application will require a high rate of API calls, please contact support@dwolla.com for further assistance.
 
+### IP Allowlist
+
+IP allowlisting enables you to define IP addresses from which you want to allow access to the Dwolla API. This helps to prevent unauthorized networks from performing actions on behalf of your application.
+
+**Things to remember:**
+
+- The IP Allowlist option is made available once an application has completed the application approval process.
+- If you are logged in as a sub-user, you will need Edit permissions for Applications to be able to edit the IP allowlist.
+
+To set up IP allowlisting navigate to the Applications tab in the Dashboard. Locate your application for which you want to specify a list of IP addresses, then click on Allow List. You can then specify an IP address or a range of IP addresses and add a description for that entry.
```

Comment:
For the sake of disambiguation, it may be worth noting here that "range of IP addresses" means a CIDR range.